components/BarList/index.js

```diff
@@ -8,10 +8,11 @@ const matchesInBar = barsWithMatches.map((bar) => (
 {...bar, 
 matches: matches.filter((match) => bar.matches.includes(match.id)).map((team) => ({homeTeam: team.homeTeam, awayTeam: team.     awayTeam})) 
 }))
+
     return(
         <BarsUl>
-            {matchesInBar.map((barWithMatch) => (
-                <BarCard key={barWithMatch.id} barWithMatch={barWithMatch}/>
+            {matchesInBar.map((barWithMatch, index) => (
+                <BarCard key={`${barWithMatch.id}-${index}`} barWithMatch={barWithMatch}/>
```

Comment:
Me neither, I think it was because when I tested, sometimes a BarCard also included a match with the same id as the bar. Therefore my compiler told me to change it and I searched for a solution to make it really really unique 😄 
When I would work with an API, I will change it back to only id, but in this case I wanted to make sure to have no error messages.